internal/cmd/iq.go

```diff
@@ -249,8 +249,13 @@ func auditWithIQServer(purls []string) error {
 
 	if res.PolicyAction != "Failure" {
 		logLady.WithField("res", res).Debug("Successful in communicating with IQ Server")
-		fmt.Println("Wonderbar! No policy violations reported for this audit!")
-		fmt.Println("Report URL: ", res.ReportHTMLURL)
+		if res.PolicyAction == "Warning" {
```

Comment:
Personally, I'd switch to a `switch` on this, over an `if else`, I generally do that any time I'm over two values.

Reply:
Thanks for the reminder! 8edd102